pkgs/development/python-modules/tifffile/default.nix

```diff
@@ -0,0 +1,25 @@
+{lib, stdenv, fetchFromGitHub, python, buildPythonPackage, isPyPy, isPy27, numpy, nose,
+ enum34, pythonOlder, futures}:
+
+buildPythonPackage rec {
+  pname = "tifffile";
+  rev = "v0.13.0";
+  name = "${pname}-${rev}";
+
+  src = fetchFromGitHub {
+    owner = "blink1073";
+    repo = "tifffile";
+    inherit rev;
+    sha256 = "0adm1zf3b4gf8yjriidjnl9abcycqiy5bzannwyb8rcbh3jwdbzv";
+  };
+
+  buildInputs = [ nose ];
+    
+  propagatedBuildInputs = [ numpy ] ++ lib.optional isPy27 futures ++ lib.optional (pythonOlder "3.0") enum34;
```

Comment:
Would `[ numpy ] ++ lib.optionals (!isPy3k) [ futures enum34 ]` maybe be more suitable?

Reply:
This expression is equal this piece of code in [setup.py](https://github.com/blink1073/tifffile/blob/f8c0f4d6addbee2a90060f87fb19458b7b06a286/setup.py):
```python
        'enum34;python_version<"3.0"',
        'futures; python_version == "2.7"'
```